nautobot/docs/administration/nautobot-server.md

```diff
@@ -0,0 +1,476 @@
+# The Nautobot Server Command
+
+Nautobot includes a command-line (CLI) management utility called `nautobot-server`, that is used as a single entrypoint for common administrative tasks.
+
+## Background
+
+For those familiar with Django applications, this CLI utility works exactly as a project's `manage.py` script would, except that it comes bundled with the Nautobot code and therefore it gets automatically installed in the `bin` directory of your application environment.
+
+!!! important
+    Since Nautobot is a Django application, there are a number of built-in management commands that will not be covered in this document. Please see the [official Django documentation on management commands](https://docs.djangoproject.com/en/stable/ref/django-admin/#available-commands) for more information.
+
+## Getting Help
+
+To see all available management commands:
+
+```no-highlight
+$ nautobot-server help
+```
+
+All management commands have a `-h/--help` flag to list all available arguments for that command, for example:
+
+```no-highlight
+$ nautobot-server migrate --help
+```
+
+## Available Commands
+
+### `collectstatic`
+
+`nautobot-server collectstatic`
+
+Collect static files into [`STATIC_ROOT`](../../configuration/optional-settings/#static_root).
+
+```no-highlight
+$ nautobot-server collectstatic
+
+965 static files copied to '/opt/nautobot/static'.
+```
+
+!!! note
+    This is a built-in Django command. Please see the [official documentation on `collectstatic`](https://docs.djangoproject.com/en/stable/ref/django-admin/#collectstatic) for more information.
+
+### `createsuperuser`
+
+`nautobot-server createsuperuser`
+
+Creates a superuser account that has all permissions. 
+
+```no-highlight
+$ nautobot-server createsuperuser
+Username (leave blank to use 'jathan'): example
```

Comment:
Would prefer to have actual copy/paste from the terminal to avoid human errors manipulating data after the fact.

Reply:
LOL yeah.

Reply:
This is a copy/paste tho. We could just leave it out? (It will default to your local username).

Reply:
got it, it always says root for me :) Did not realize it was my Docker username.